.github/linters/sun_checks.xml

```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0"?>
+<!--
+  ~ ********************************************************************************
+  ~ Copyright (c) 2024 Contributors to the Eclipse Foundation
+  ~
+  ~ See the NOTICE file(s) where distributed with this work for additional
+  ~ information regarding copyright ownership.
+  ~
+  ~ This program and the accompanying materials are made
+  ~ available under the terms of the Apache Software License 2.0
+  ~ which is available at https://www.apache.org/licenses/LICENSE-2.0.
+  ~
+  ~ SPDX-License-Identifier: Apache-2.0
+  ~ ********************************************************************************
+-->
+
```

Comment:
Appropriate on these Github config files?

Reply:
I think it's okay, it's still our code that we've written